bin/setup_base

```diff
@@ -61,7 +61,7 @@ echo Adding username to docker group...
 sudo groupadd docker || true
 sudo usermod -aG docker $user
 
-DEF_IFACE=`sudo route -n | fgrep UG | awk '{print $8}'`
+DEF_IFACE=`sudo route -n | fgrep UG | head -1 | awk '{print $8}'`
```

Comment:
We need to make sure this is the right logic.  How is it happening that there's two gateways?  The behaviour in this situation is undefined (which one is right?) so it usually an indicator of a misconfigured system.

Reply:
updated. PTAL